README.rst

```diff
@@ -0,0 +1,60 @@
+===========
+aiocfscrape
+===========
+
+A simple async Python module to bypass Cloudflare\'s anti-bot page.
+Based on aiohttp ClientSession. Solution was inherited from `cfscrape <https://github.com/Anorov/cloudflare-scrape>`_
+module.
+
+You could use it eg. with Python 3 and `asyncio <https://docs.python.org/3/library/asyncio-dev.html>`_
+for concurrent crawling of web resources protected with CloudFlare.
+
+
+.. contents:: Table of Contents
+
+
+Installation
+============
+
+Install with pip
+
+.. code:: sh
+
+    pip install aiocfscrape2
```

Comment:
Are you still going to use a new package name on PyPi?

Reply:
oops, is a mistake